lib/workers/repository/process/vulnerabilities.ts

```diff
@@ -123,20 +155,19 @@ export class Vulnerabilities {
       packageName = packageName.toLowerCase().replace(regEx(/[_.-]+/g), '-');
     }
 
-    const packageRules: PackageRule[] = [];
     try {
-      const vulnerabilities = await this.osvOffline?.getVulnerabilities(
+      const osvVulnerabilities = await this.osvOffline?.getVulnerabilities(
```

Comment:
this rename is also causing a lot of changes

Reply:
IMO this is necessary as else it is obvious which type of vulnerability is referenced here our internal interface or OsvVulnerabilities. 

I prefer to keep it for better readability.

Reply:
sure, i just thought to have the rename in a seperate PR. but it's ok to leave here.